README.md

```diff
@@ -22,8 +23,8 @@ instance:
   VERSION=0.1.0
   NAMESPACE=default
   GITEA_ENDPOINT="http://gitea-http.${NAMESPACE}:3000/"
-  GITEA_ADMIN_USERNAME=#Define a username for the admin
```

Comment:
Is there any special reason that the `GITEA_ENDPOINT` variable was removed? I would prefer the definition of the endpoint up here and use it in the helm install later on